PWGGA/GammaConv/AliPrimaryPionCuts.cxx

```diff
@@ -1511,6 +1551,11 @@ Bool_t AliPrimaryPionCuts::SetMassCut(Int_t massCut){
             fDoMassCut = kTRUE;
             fMassCut = 0.520;
             break;
+        case 22: //m overload mass cut for chi2 of vParticle
```

Comment:
shouldnt this be case 23 as case 22 already exists? This way the old case 22 will never be reached